pkg/kubelet/volumes.go

```diff
@@ -263,23 +251,27 @@ func (kl *Kubelet) getPodVolumesFromDisk() map[string]cleanerTuple {
 	return currentVolumes
 }
 
+// newVolumeMounterFromPlugins attempts to find a plugin by volume spec and then create a physicalMounter from
```

Comment:
Strike this comment; there's a lot more to say about this method than is in-scope for this PR